assets/js/modules/analytics/components/dashboard/DashboardSearchVisitorsWidget.stories.js

```diff
@@ -0,0 +1,193 @@
+/**
+ * DashboardSearchVisitorsWidget Component Stories.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { CORE_USER } from '../../../../googlesitekit/datastore/user/constants';
+import { STORE_NAME } from '../../datastore/constants';
+import { provideModules, provideSiteInfo } from '../../../../../../tests/js/utils';
+import { withWidgetComponentProps } from '../../../../googlesitekit/widgets/util';
+import { provideAnalyticsMockReport } from '../../util/data-mock';
+import WithRegistrySetup from '../../../../../../tests/js/WithRegistrySetup';
+import DashboardSearchVisitorsWidget from './DashboardSearchVisitorsWidget';
+
+const reportOptions = [
+	{
+		compareStartDate: '2020-07-14',
+		compareEndDate: '2020-08-10',
+		startDate: '2020-08-11',
+		endDate: '2020-09-07',
+		dimensionFilters: { 'ga:channelGrouping': 'Organic Search' },
+		dimensions: [ 'ga:channelGrouping' ],
+		metrics: [
+			{
+				expression: 'ga:users',
+				alias: 'Total Users',
+			},
+		],
+	},
+	{
+		startDate: '2020-08-11',
+		endDate: '2020-09-07',
+		dimensionFilters: { 'ga:channelGrouping': 'Organic Search' },
+		dimensions: [ 'ga:date', 'ga:channelGrouping' ],
+		metrics: [
+			{
+				expression: 'ga:users',
+				alias: 'Users',
+			},
+		],
+	},
+];
+const currentEntityURL = 'https://www.example.com/example-page/';
+const reportOptionsWithEntity = reportOptions.map( ( options ) => {
+	return {
+		...options,
+		url: currentEntityURL,
+	};
+} );
+
+const WidgetWithComponentProps = withWidgetComponentProps( 'widget-slug' )( DashboardSearchVisitorsWidget );
```

Comment:
Maybe it would be more appropriate for this to be the real widget slug instead? Not really an issue though.

Reply:
My preference is to use dummy values when it isn't necessary/significant. It's also one less place to change if the slug were to change 😄